src/xenia/ui/imgui_drawer.cc

```diff
@@ -142,10 +142,12 @@ void ImGuiDrawer::Initialize() {
   InitializeFonts();
 
   auto& style = ImGui::GetStyle();
-  style.ScrollbarRounding = 0;
-  style.WindowRounding = 0;
-  style.TabRounding = 0;
-  style.Colors[ImGuiCol_Text] = ImVec4(0.89f, 0.90f, 0.90f, 1.00f);
+  style.ScrollbarRounding = 8.0f;
+  style.WindowRounding = 8.0f;
+  style.PopupRounding = 8.0f;
+  style.TabRounding = 8.0f;
+
+  style.Colors[ImGuiCol_Text] = ImVec4(1.00f, 1.00f, 1.00f, 1.00f);
```

Comment:
I think that having it completely white is a bad idea, normally text is slightly grayed for few reasons